pyinfra_xdcstore/xstore-update.sh

```diff
@@ -6,7 +6,7 @@ set -e
 
 cd xdcget
 git pull --autostash --rebase origin main
-pip install --user .
+pip install --break-system-packages --user .
```

Comment:
To ignore this error:

```
Sep 26 17:18:07 bomba xstore-update.sh[2706]: error: externally-managed-environment
Sep 26 17:18:07 bomba xstore-update.sh[2706]: × This environment is externally managed
Sep 26 17:18:07 bomba xstore-update.sh[2706]: ╰─> To install Python packages system-wide, try apt install
Sep 26 17:18:07 bomba xstore-update.sh[2706]:     python3-xyz, where xyz is the package you are trying to
Sep 26 17:18:07 bomba xstore-update.sh[2706]:     install.
Sep 26 17:18:07 bomba xstore-update.sh[2706]:     
Sep 26 17:18:07 bomba xstore-update.sh[2706]:     If you wish to install a non-Debian-packaged Python package,
Sep 26 17:18:07 bomba xstore-update.sh[2706]:     create a virtual environment using python3 -m venv path/to/venv.
Sep 26 17:18:07 bomba xstore-update.sh[2706]:     Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
Sep 26 17:18:07 bomba xstore-update.sh[2706]:     sure you have python3-full installed.
Sep 26 17:18:07 bomba xstore-update.sh[2706]:     
Sep 26 17:18:07 bomba xstore-update.sh[2706]:     If you wish to install a non-Debian packaged Python application,
Sep 26 17:18:07 bomba xstore-update.sh[2706]:     it may be easiest to use pipx install xyz, which will manage a
Sep 26 17:18:07 bomba xstore-update.sh[2706]:     virtual environment for you. Make sure you have pipx installed.
Sep 26 17:18:07 bomba xstore-update.sh[2706]:     
Sep 26 17:18:07 bomba xstore-update.sh[2706]:     See /usr/share/doc/python3.11/README.venv for more information.
Sep 26 17:18:07 bomba xstore-update.sh[2706]: note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by>
Sep 26 17:18:07 bomba xstore-update.sh[2706]: hint: See PEP 668 for the detailed specification.
```